packages/tocco-util/src/form/formBuilder.js

```diff
@@ -135,19 +136,30 @@ export default (
   }
 
   const createLayoutComponent = (field, type, key, traverser) => {
-    if (type === layoutTypes.HORIZONTAL_BOX || type === layoutTypes.VERTICAL_BOX) {
-      const alignment = type === layoutTypes.HORIZONTAL_BOX ? 'horizontal' : 'vertical'
-      const children = traverser()
+    let elements = traverser()
 
-      if (children == null || (Array.isArray(children) && children.every(e => e == null))) {
-        return null
-      }
+    if (Array.isArray(elements)) {
+      elements = elements.filter(el => el)
+    }
 
-      return (
-        <LayoutBox key={key} label={field.label} alignment={alignment}>
-          {children}
-        </LayoutBox>
-      )
+    if (!elements || (Array.isArray(elements) && elements.every(e => e == null))) {
```

Comment:
` && elements.every(e => e == null)`

makes no sense here because on line 142 you filtered out any element that is null.

`&& elements.length === 0`